src/main/java/step2/model/Lottos.java

```diff
@@ -37,7 +37,7 @@ public long getWinningCount (LottoRank rank, LottoWinning lottoWinning) {
     }
 
     public void addLottos(Lottos lottos) {
-        this.lottos.addAll(lottos.stream().collect(toList()));
+        this.lottos.addAll(0, lottos.stream().collect(toList()));
```

Comment:
특정 인덱스부터 복사하는것이 아니라면 인덱스를 사용할 필요는 없어보입니다 :)
```suggestion
        this.lottos.addAll(lottos.stream().collect(toList()));
```